pilot/pkg/networking/core/v1alpha3/gateway.go

```diff
@@ -424,7 +424,7 @@ func buildGatewayListenerTLSContext(
 
 	tls := &auth.DownstreamTlsContext{
 		CommonTlsContext: &auth.CommonTlsContext{
-			AlpnProtocols: util.ALPNHttp,
+			AlpnProtocols: util.ALPNDownstream,
```

Comment:
Why changing the TLS context for gateway?

Reply:
Thanks for pointing.. removing